### DISCUSSION TITLE
imdb_list type filter

### DISCUSSION THREAD

**Discussion by Nemesis7** [Flexget/Flexget#3711] [Ideas]
Would it not be nice to be able to filter on `type` (e.g. movie and show) when using `imdb_list`, similar to using a `plex_watchlist`?

**Comment by BrutuZ** [comment]
Would be even nicer if IMDB had an open API

**Reply by Nemesis7** [reply]
I know mate, but we already have the imdb_type / type which we can use to filter the results. This works in other tasks, but not nicely with multiple list_match entries:
```yaml
      if:
        - imdb_type == 'series': accept
```
Filtering should be possible for both `imdb_watchlist` and `imdb_list` with the current entry (meta)data

**Comment by gazpachoking** [comment]
PRs would be welcome to add this field.
